modules/editor-one/assets/js/admin-menu/classes/sidebar-menu-handler.js

```diff
@@ -28,10 +29,10 @@ export class SidebarMenuHandler {
 	}
 
 	activateElementorMenu() {
-		this.elementorMenu.classList.remove( 'wp-not-current-submenu' );
-		this.elementorMenu.classList.add( 'wp-has-current-submenu', 'wp-menu-open', 'selected' );
+		this.elementorHomeMenu.classList.remove( 'wp-not-current-submenu' );
```

Comment:
Also for other usages
```suggestion
		if (!this.elementorHomeMenu) {
				return;
		}
		this.elementorHomeMenu.classList.remove( 'wp-not-current-submenu' );
```

Reply:
@ManorHazaz this is not needed if you look at the entire file. Even before the refactoring you asked me this would only be called if we have a menu.